assets/lokomotive-kubernetes/bootkube/versions.tf

```diff
@@ -6,5 +6,6 @@ terraform {
     local    = "~> 1.2"
     template = "~> 2.1"
     tls      = "~> 2.0"
+    random   = "~> 2.2"
```

Comment:
I think this should be added together with it's being used. Otherwise it's a commit with no context.

Reply:
It is used in packet, aws and bare-metal all of them.